.gitignore

```diff
@@ -250,23 +250,16 @@ paket-files/
 .idea/
 *.sln.iml
 
-# Visual Studio Code
-.vscode/launch.json
-.vscode/tasks.json
-
-# launchSettings.json
-launchSettings.json
-
 # Node and .NET Core runtime images dockerfiles, that are auto generated
 images/runtime/node/*/Dockerfile
 images/runtime/dotnetcore/*.*/
 images/runtime/python/*.*/
 images/runtime/php/*.*/
 
-# Go packages, if Oryx is added to GOPATH (which it should)
-pkg/
-
 # Go packages from GitHub
-src/github.com/
-src/startupscriptgenerator/vendor/
-src/golang.org/
\ No newline at end of file
+src/startupscriptgenerator/src/github.com/
```

Comment:
these should go under `src/startupscriptgenerator/src/vendor`, no?

Reply:
also, it feels a bit odd to have `src` under `src`

Reply:
> these should go under src/startupscriptgenerator/src/vendor, no?
 
but I did not see that happening in my workspace...let me investigate further

> also, it feels a bit odd to have src under src

yes, I agree, but that is the nature of our repo where we have different platforms (.net core and go)..any suggestions?

Reply:
Regarding vendor folder, yes you are right...i was earlier doing a `go get` which created these folders `github.com` etc. but if we use `dep` tool, it puts under a vendor folder. Will update